### PR TITLE
[fix] backend peerDependencies instead of dependencies npm 2.X

### PR DESCRIPTION
**Pull request by emmoistner**

### CODE REVIEW

File: packages/eslint-config-emplify-backend/test/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+'use strict';
```
Comment (davidharting) [inline packages/eslint-config-emplify-backend/test/index.js]: Get rid of this file?

File: packages/eslint-config-emplify-backend/test/package.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (davidharting) [inline packages/eslint-config-emplify-backend/test/package.json]: get rid of this file?

File: packages/eslint-config-emplify-backend/test/.eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: 'eslint-config-emplify-backend'
```
Comment (davidharting) [inline packages/eslint-config-emplify-backend/test/.eslintrc.js]: get rid of this file?

**Comment by emmoistner** [comment]
Added local testing instructions. Explains the testing directory

**Pull request merged**